costmap_cspace/CMakeLists.txt

```diff
@@ -23,15 +21,19 @@ catkin_package(
   CATKIN_DEPENDS ${CATKIN_DEPENDS}
 )
 
-
 set(CMAKE_CXX_STANDARD 14)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
-include_directories(include ${catkin_INCLUDE_DIRS})
+include_directories(include ${catkin_INCLUDE_DIRS} ${CMAKE_CURRENT_SOURCE_DIR}/include/costmap_cspace) #add header files in costmap_cspace
 
+# Add library for costmap_3d_layers.cpp
+add_library(costmap_3d_layers_lib src/costmap_3d_layers.cpp)
```

Comment:
I think `_lib` is not very common for linux library file name and this will result `lib/libcostmap_3d_layers_lib.so`.
So, `costmap_3d_layers` would be better if there's no name conflict.

Reply:
Thank you for your suggestions. I will make changes and PR the corrections.